### PR TITLE
Logo doc

### PR DESCRIPTION
**Pull request by SimpleVlad**

### CODE REVIEW

File: curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/logo.md [hunk 1]
```diff
@@ -86,14 +85,15 @@ tests:
...
-    cv.ellipse(mat, new cv.Point(220, 170), new cv.Size(50, 50), 300.0, 0.0, 300.0, new cv.Scalar(0, 0, 255, 255), cv.FILLED);
-    cv.ellipse(mat, new cv.Point(80, 170), new cv.Size(50, 50), 0.0, 0.0, 300.0, new cv.Scalar(0, 255, 0, 255), cv.FILLED);
-    cv.ellipse(mat, new cv.Point(150, 60), new cv.Size(50, 50), 120.0, 0.0, 300.0, new cv.Scalar(255, 0, 0, 255), cv.FILLED);
-  
-    cv.circle(mat, new cv.Point(220, 170), 25, new cv.Scalar(255, 255, 255, 255), cv.FILLED);
-    cv.circle(mat, new cv.Point(80, 170), 25, new cv.Scalar(255, 255, 255, 255), cv.FILLED);
-    cv.circle(mat, new cv.Point(150, 60), 25, new cv.Scalar(255, 255, 255, 255), cv.FILLED);
+    let angle = [300.0, 0.0, 120.0];
+    let color = [new cv.Scalar(0, 0, 255, 255),new cv.Scalar(0, 255, 0, 255),new cv.Scalar(255, 0, 0, 255)];
+    let startPoin = [new cv.Point(220, 170),new cv.Point(80, 170), new cv.Point(150, 60)];
```
Comment (dkurt) [inline curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/logo.md]: angles, colors, etc.

File: curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/logo.md [hunk 2]
```diff
@@ -86,14 +85,15 @@ tests:
...
+
+
+    for (let i=0; i<3; ++i){
+        cv.ellipse(mat,startPoin[i], new cv.Size(50, 50), angle[i], 0.0, 300.0, color[i], cv.FILLED);
+        cv.circle(mat,startPoin[i], 25, new cv.Scalar(255, 255, 255, 255), cv.FILLED);
+    }
```
Comment (dkurt) [inline curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/logo.md:103]: Let's define constants:
```
let circleRadius = 25;
let ellipseAxes = new cv.Size(50, 50);
```
and so on

Comment (SimpleVlad) [reply curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/logo.md:103]: work in progress)))I wanted to ask about it

**Comment by SimpleVlad** [comment]
In <code>Solution</code> stay a last iteration of logo

File: curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/logo.md [hunk 3]
```diff
@@ -46,16 +46,22 @@ tests:
...
-  
-    cv.ellipse(mat, new cv.Point(220, 170), new cv.Size(50, 50), 300.0, 0.0, 300.0, new cv.Scalar(0, 0, 255, 255), cv.FILLED);
-    cv.ellipse(mat, new cv.Point(80, 170), new cv.Size(50, 50), 0.0, 0.0, 300.0, new cv.Scalar(0, 255, 0, 255), cv.FILLED);
-    cv.ellipse(mat, new cv.Point(150, 60), new cv.Size(50, 50), 120.0, 0.0, 300.0, new cv.Scalar(255, 0, 0, 255), cv.FILLED);
-  
-    cv.circle(mat, new cv.Point(220, 170), 25, new cv.Scalar(255, 255, 255, 255), cv.FILLED);
-    cv.circle(mat, new cv.Point(80, 170), 25, new cv.Scalar(255, 255, 255, 255), cv.FILLED);
-    cv.circle(mat, new cv.Point(150, 60), 25, new cv.Scalar(255, 255, 255, 255), cv.FILLED);
+    let angles = [300.0, 0.0, 120.0];
+    let anglesEnd = [600.0, 300.0, 420.0];
```
Comment (dkurt) [inline curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/logo.md]: mb startAngles and endAngles?

Please make them in range [0, 360)

File: curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/logo.md [hunk 4]
```diff
@@ -46,16 +46,22 @@ tests:
...
-    cv.ellipse(mat, new cv.Point(80, 170), new cv.Size(50, 50), 0.0, 0.0, 300.0, new cv.Scalar(0, 255, 0, 255), cv.FILLED);
-    cv.ellipse(mat, new cv.Point(150, 60), new cv.Size(50, 50), 120.0, 0.0, 300.0, new cv.Scalar(255, 0, 0, 255), cv.FILLED);
-  
-    cv.circle(mat, new cv.Point(220, 170), 25, new cv.Scalar(255, 255, 255, 255), cv.FILLED);
-    cv.circle(mat, new cv.Point(80, 170), 25, new cv.Scalar(255, 255, 255, 255), cv.FILLED);
-    cv.circle(mat, new cv.Point(150, 60), 25, new cv.Scalar(255, 255, 255, 255), cv.FILLED);
+    let startAngles = [240.0, 0.0, -240.0];
+    let endAngles = [-60.0, 300.0, 60.0];
+    let colors = [new cv.Scalar(0, 0, 255, 255), new cv.Scalar(0, 255, 0, 255),new cv.Scalar(255, 0, 0, 255)];
+    let startPoints = [new cv.Point(220, 170), new cv.Point(80, 170), new cv.Point(150, 60)];
```
Comment (dkurt) [inline curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/logo.md]: centers?

File: curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/logo.md (hunk 1 above)
Comment (dkurt) [reply curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/logo.md]: `startPoin` -> `centers`

File: curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/logo.md [hunk 5]
```diff
@@ -78,16 +87,21 @@ tests:
...
+    let endAngles = [-60.0, 300.0, 60.0];
+    let colors = [new cv.Scalar(0, 0, 255, 255), new cv.Scalar(0, 255, 0, 255),new cv.Scalar(255, 0, 0, 255)];
+    let centers = [new cv.Point(220, 170), new cv.Point(80, 170), new cv.Point(150, 60)];
+    let circleRadius = 25;
+    let whiteColor = new cv.Scalar(255, 255, 255, 255);
+    let ellipseAxes = new cv.Size(50, 50);
+
+    for (let i=0; i<3; ++i){
+        cv.ellipse(mat,centers[i], ellipseAxes, 0.0, 0.0, 300.0, color[i], cv.FILLED);
+        cv.circle(mat,centers[i], 25, new cv.Scalar(255, 255, 255, 255), cv.FILLED);
```
Comment (dkurt) [inline curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/logo.md]: What about arguments? Are they correct?

File: curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/logo.md [hunk 6]
```diff
@@ -7,21 +7,18 @@ videoUrl:
...
-First, you'll start by building a simple web page using HTML. You can edit <code>code</code> in your <code>code editor</code>, which is embedded into this web page.
-Do you see the code in your code editor that says <code>&#60;h1&#62;Hello&#60;/h1&#62;</code>? That's an HTML <code>element</code>.
-Most HTML elements have an <code>opening tag</code> and a <code>closing tag</code>.
-Opening tags look like this:
-<code>&#60;h1&#62;</code>
-Closing tags look like this:
-<code>&#60;/h1&#62;</code>
-The only difference between opening and closing tags is the forward slash after the opening bracket of a closing tag.
-Each challenge has tests you can run at any time by clicking the "Run tests" button. When you pass all tests, you'll be prompted to submit your solution and go to the next coding challenge.
+Sometimes we need to select any objects at the image. OpenCV has drawing functions. In this tutorial, we’ll look at features like cv.ellipse() and cv.circle(). The main aim of this lesson is show how use this function and draw openCV logo.
```
Comment (dkurt) [inline curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/logo.md]: `Sometimes we need to select any objects at the image`

What is object selection? I think it's OK to start with `OpenCV has drawing functions which let you draw simple geometric primitives.`

Comment (dkurt) [inline curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/logo.md]: it to show how to use

Comment (dkurt) [inline curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/logo.md]: we'll -> we will

(it just more convenient for technical documents)

Comment (dkurt) [inline curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/logo.md]: features -> methods

function -> functions

File: curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/logo.md [hunk 7]
```diff
@@ -7,21 +7,18 @@ videoUrl:
...
+
+<code>ellipse(img, center, axes, angle, startAngle, endAngle, color)</code> where <code>img</code> is an image where the circle is drawn, <code>center</code> - Center of the circle, <code>angle</code> is ellipse rotation angle in degrees, <code>startAngle</code> is starting angle of the elliptic arc in degrees, <code>endAngle</code> is ending angle of the elliptic arc in degrees,<code>color</code> is a color of object, and <code>thickness</code> is thickness of the ellipse arc outline, if positive. Otherwise, this indicates that a filled ellipse sector is to be drawn.
```
Comment (dkurt) [inline curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/logo.md]: ellipse is drawn?

File: curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/logo.md [hunk 8]
```diff
@@ -7,21 +7,18 @@ videoUrl:
...
-Closing tags look like this:
-<code>&#60;/h1&#62;</code>
-The only difference between opening and closing tags is the forward slash after the opening bracket of a closing tag.
-Each challenge has tests you can run at any time by clicking the "Run tests" button. When you pass all tests, you'll be prompted to submit your solution and go to the next coding challenge.
+OpenCV has drawing functions which let you draw simple geometric primitives. In this tutorial, we will look at methods like cv.ellipse() and cv.circle(). The main aim of this lesson is to show how to use this functions and draw openCV logo.
+
+<code>ellipse(img, center, axes, angle, startAngle, endAngle, color)</code> where <code>img</code> is an image where the elips is drawn, <code>center</code> - Center of the elips, <code>angle</code> is ellipse rotation angle in degrees, <code>startAngle</code> is starting angle of the elliptic arc in degrees, <code>endAngle</code> is ending angle of the elliptic arc in degrees,<code>color</code> is a color of object, and <code>thickness</code> is thickness of the ellipse arc outline, if positive. Otherwise, this indicates that a filled ellipse sector is to be drawn.
```
Comment (dkurt) [inline curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/logo.md]: elips -> ellipse

Comment (dkurt) [reply curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/logo.md]: Center - center

File: curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/logo.md [hunk 9]
```diff
@@ -7,21 +7,18 @@ videoUrl:
...
+
+<code>circle(img, center, radius, color)</code> where <code>img</code> is an image where the circle is drawn, <code>center</code> - Center of the circle, <code>radius</code> is a circle's radius, and <code>color</code> is a color of object.
 </section>
 
 ## Instructions
 <section id='instructions'>
-I don't know what is happaning
+
+You should use <code>cv.ellipse()</code> to draw a elliptical figur with the given parameters in <code>for</code>>.
+Also we need to use <code>cv.circle()</code> to draw a circle in <code>for</code> too. We need it for drow a centers our figurs.
```
Comment (dkurt) [inline curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/logo.md]: We need it for drow a centers ->  We need it to draw centers

File: curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/logo.md [hunk 10]
```diff
@@ -7,21 +7,18 @@ videoUrl:
...
+
+<code>circle(img, center, radius, color)</code> where <code>img</code> is an image where the circle is drawn, <code>center</code> - center of the circle, <code>radius</code> is a circle's radius, and <code>color</code> is a color of object.
 </section>
 
 ## Instructions
 <section id='instructions'>
-I don't know what is happaning
+
+You should use <code>cv.ellipse()</code> to draw a elliptical figur with the given parameters in <code>for</code>>.
+Also we need to use <code>cv.circle()</code> to draw a circle in <code>for</code> too. We need it to draw centers our figurs.
```
Comment (dkurt) [inline curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/logo.md]: centers of our figures

figurs -> figures
figur -> figure

here and in other places

**Review by dkurt — APPROVED** [review]
:+1: LGTM

**Pull request merged**